plugins/module_utils/network/eos/config/acls/constants.py

```diff
@@ -0,0 +1,191 @@
+# constants.py
+
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+
+PORT_NUMBERS = [
```

Comment:
A better place for this would be utils.py https://github.com/ansible-collections/arista.eos/blob/main/plugins/module_utils/network/eos/utils/utils.py